.claude/skills/create-pr/SKILL.md

```
@@ -0,0 +1,172 @@
+## Step 0. 현재 상태 파악
+
+아래 명령어를 **동시에** 실행합니다.
+
+```bash
+git branch --show-current          # 현재 브랜치명
+git config user.name               # assignee용 사용자명
+gh api user --jq .login            # GitHub 로그인 ID (assignee 설정에 사용)
+```
+
+### 브랜치명에서 이슈 번호 추출
+
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)